src/main/resources/hudson/plugins/sshslaves/SSHConnector/config.jelly

```diff
@@ -40,5 +40,9 @@
           <f:textbox name="retryWaitTime" />
       </f:entry>
 
+      <f:entry title="${%Disable TCP_NODELAY flag on the SSH connection}" field="disableTcpNoDelay">
+          <f:checkbox name="disableTcpNoDelay" />
+      </f:entry>
```

Comment:
Does this retain the prior default? I'm not familiar enough with Jenkins jelly components to be sure how that works.

Reply:
yes, because the default value of a boolean is false